src/CoreBundle/CronJob/CronJobSchedulerInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace ChameleonSystem\CoreBundle\CronJob;
+
+interface CronJobSchedulerInterface
```

Comment:
I am open to alternative suggestions. Given a lack of a better option I would say that the name matches the job close enough (it does provide the next planned execution - which is somewhat like a scheduler).

Reply:
CronjobService would be innocent enough.

It does (should) do everything that scheduler and data access don't do.

Reply:
Cronjob Service should take care of more than just the scheduling part - which is exactly what the class is currently doing. Granted, there are external classes also involved in the scheduling currently - so I kind of agree that scheduler is not super precises. But renaming it to CronjobService is worse in my opinion since that name really tells us nothing about the responsibilities the service has.